src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,172 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node head;
+    private Node tail;
```

Comment:
Prametrize it with generics
```suggestion
    private Node<T> head;
    private Node<T> tail;
```